packages/storybook/stories/menu-button.mdx

```diff
@@ -0,0 +1,52 @@
+import { MenuButton } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Stories, Story, Canvas } from '@storybook/blocks';
+import * as MenuButtonStories  from './menu-button.stories';
+
+<Meta of={MenuButtonStories} />
+
+# Menu Button
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+Menu Button allows users to open a menu with a list of interactive options (actions) [1].
+<Canvas of={MenuButtonStories.Secondary} />
+
+## Usage
+### When to use 
+- Use for actions that will change the application state (either front or back end) [2].
+- Use when you need to show numerous options but have limited screen space available to display them in separate buttons [1].
+
+### When not to use 
+- Do not use when you have to bring the user somewhere else into or outside the application, use dropdown navigation instead [2].
```

Comment:
on pourrait simplifier "_Do not use for navigation purposes, use dropdown navigation instead_" Tu en penses quoi?